README.rst

```diff
@@ -5,14 +5,15 @@
 Object-Relational Mapping for DVC
 ---------------------------------
 
-The ZnTrack [zɪŋk træk] package provides a Python ORM for DVC.
+The ZnTrack [zɪŋk træk] package provides an easy-to-use interface for DVC directly from within Python.
```

Comment:
I think we should discuss how to best advertise the package. I would position ZnTrack as an entry point to dvc and not the other way around. This would discussing ZnTrack as a package for parameter tracking which builds on DVC. You do not need any knowledge of DVC to use the package so I would put that elsewhere. In it's current state it looks like an aid for people using DVC but really it is for anyone and the package itself uses DVC.

Reply:
Is the new version satisfactory?

Reply:
I like the changes you made! In general I would like to optimize the Readme and add a better example but the new introducion is a good first step! Thank you.